### PR TITLE
Call `track.lyrics.get` with `track_id` to get lyrics

### PR DESCRIPTION
**Pull request by greenerchen**

### CODE REVIEW

File: Tests/MusixmatchAPITests/MusixmatchAPITests.swift [hunk 1]
```diff
@@ -10,7 +10,7 @@ import XCTest
 
 final class MusixmatchAPITests: XCTestCase {
 
-    func test_search_withArtistNamesAndTitle_getSongLyrics() async throws {
+    func test_trackSearch_withArtistNamesAndTitle_getSongLyrics() async throws {
```
Comment (greenerchen) [inline Tests/MusixmatchAPITests/MusixmatchAPITests.swift:13]: Add edge cases like:
- wrong combination of artist and track
- latest/nonexistent track that hasn't been searchable
